### ISSUE TITLE
body-parser Usage

### ISSUE THREAD

**Issue by sdoward** [lukeautry/tsoa#1231]
I have recently done some work to integrate Tsoa on a codebase. The docs show that we have to import `body-parser` dependency but it the docs don't explain why. Tsoa seems to function without. Is necessary? Is there some reason Tsoa doesn't automatically include this dependency?

I am happy to update the docs if need be

**Comment by FrozenKiwi** [comment]
I'm not 100% sure, but in my code base I have
```
// Use body parser to read sent json payloads
// otherwise nothing is recieved in body
app.use(bodyParser.urlencoded({extended: true}));
app.use(bodyParser.json());
```

**Comment by b1tw0rker** [comment]
Actually, tsoa developers may have some focus onit. body-parser is not needed using express 4.16+.
body-parsing is implemented in express 4.1.6+. So it should be testes using tsoa without body-parser.

Useful article to this topic can be found here:
https://medium.com/@mmajdanski/express-body-parser-and-why-may-not-need-it-335803cd048c
